tests/benchmark/Dockerfile.in

```diff
@@ -1,6 +1,15 @@
-FROM ${FULL_BASE_IMAGE}
+FROM gcr.io/google-appengine/python:latest
```

Comment:
Oh this is because I was testing it on a different docker space which does not have the tag latest. Will change it back to the variable name.